windows_service/assets/configuration/spec.yaml

```diff
@@ -64,4 +64,16 @@ files:
             value:
               type: boolean
               example: false
+          - name: collect_display_name_as_tag
+            description: |
+              Whether or not to submit the Windows Service Display Name as the `display_name` tag.
+              `display_name` will be formatted according to Datadog's tagging requirements:
```

Comment:
English my second language of course but do you think we can restate it a bit differently for clarity, e.g. Windows Service Display name will be automatically formatted according to Datadog's tagging requirements (https://docs.datadoghq.com/getting_started/tagging/#define-tags) as `display_name` tag value.

Reply:
Not sure if it worth to describe auto-formatting rules.